### PR TITLE
platform upload user list for jellyfish migration

### PR DESCRIPTION
**Pull request by jh-bate**
See https://github.com/tidepool-org/jellyfish/pull/203

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -183,6 +183,7 @@ services:
       DISCOVERY_HOST: ${TIDEPOOL_DOCKER_HAKKEN_HOST}:8000
       GATEKEEPER_SERVICE: '{"type": "static", "hosts": [{"protocol": "http", "host": "${TIDEPOOL_DOCKER_GATEKEEPER_HOST}:9123"}]}'
       MINIMUM_UPLOADER_VERSION: '2.53.0'
+      PLATFORM_UPLOAD_USERS: ''
```
Comment (darinkrauss) [inline docker-compose.yml:186]: You can drop this. We don't support this and it needs to be deleted. The whole repo needs to be cleaned up.

File: charts/tidepool/charts/jellyfish/templates/0-configmap.yaml [hunk 2]
```diff
@@ -11,4 +11,5 @@ data:
   MinimumUploaderVersion: {{ .Values.configmap.data.MinimumUploaderVersion | default "2.53.0" }}
   AuthRealm: {{ .Values.configmap.data.AuthRealm }}
   AuthUrl: {{ .Values.configmap.data.AuthUrl }}
+  PlatformUploadUsers: {{ .Values.configmap.data.PlatformUploadUsers }}
```
Comment (darinkrauss) [inline charts/tidepool/charts/jellyfish/templates/0-configmap.yaml:14]: I might call this `UploaderPlatformUserIds` and `UPLOADER_PLATFORM_USER_IDS` so it is clear.

**Review by darinkrauss — CHANGES_REQUESTED** [review]
Consider the rename, but definitely revert the `docker-compose.yaml` change (as we don't want to give any indication it is being used).

**Comment by jh-bate** [comment]
closing as now doing this in the jellyfish service itself

**Pull request closed without merging**